app/styles/sub-styles/user-card.scss

```diff
@@ -0,0 +1,31 @@
+.profile-icon {
+  width: 140px;
+  height: 140px;
+  border-radius: 50%;
+  display: flex;
```

Comment:
Why you used flex property here? You have only one item inside this flex container.

Reply:
I have used flex property for positioning the content inside the avatar circle.